test/e2e/production_test.go

```diff
@@ -1,41 +1,50 @@
+// +build elasticsearch
+
```

Comment:
I think this smoke test for ES should be moved into elasticsearch test suite - so it is equivalent to the cassandra tests.